sfm_pc/views.py

```diff
@@ -264,38 +267,36 @@ def download_zip(request):
     return resp
 
 
-class DownloadData(FormView):
```

Comment:
You can also remove the entire `sfm_pc/downloads` directory, since it is being replaced.

Reply:
I did this and it broke the tests, so now I'm having to remove all the code that was imported from here (which makes sense). 

Just want to double check if it's okay for me to remove [the `make_materialized_views` mgmt command](https://github.com/security-force-monitor/sfm-cms/blob/master/sfm_pc/management/commands/make_materialized_views.py)? I understand that command to be creating materialized views so the now obsolete download file can be more efficiently generated?

Reply:
I removed it all with this commit: dea7ec4d

Tests pass. Please take a look and let me know if this is okay. I'm especially curious if it's okay to remove the command from the migration.